meetings/2018-06-20.md

```diff
@@ -0,0 +1,104 @@
+# Node.js Foundation Technical Steering Committee (TSC) Meeting 2018-06-20
+
+## Links
+
+* **Recording**:  https://www.youtube.com/watch?v=TyrpGr_9NQc
+* **GitHub Issue**: https://github.com/nodejs/TSC/issues/553
+
+## Present
+
+* Anna Henningsen @addaleax (TSC)
+* Сковорода Никита Андреевич @ChALkeR (TSC)
+* Colin Ihrig @cjihrig (TSC)
+* Franziska Hinkelmann @fhinkel (TSC)
+* Jeremiah Senkpiel @Fishrock123 (TSC)
+* Joyee Cheung @joyeecheung (TSC)
+* Michael Dawson @mhdawson (TSC)
+* Myles Borins @MylesBorins (TSC)
+* Ali Ijaz Sheikh @ofrobots (TSC)
+* Sakthipriyan Vairamani @thefourtheye (TSC)
+
+## Agenda
+
+### Announcements
+ 
+*Extracted from **tsc-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.
+  * No announcements this week.
+
+### nodejs/build
+
+* Request for elevated permissions [#1337](https://github.com/nodejs/build/issues/1337)
+  * Joyee: request to give Jon/Rafael admin access to build infra.  Currently 5 people
+    with this level of access but they are not predictably around.  John/Rafael are handling
```

Comment:
I might be wrong but isn't it /s/John/Jon/?

Reply:
Yes, that's correct.